### DISCUSSION TITLE
A2DP Sink with MP3 Source

### DISCUSSION THREAD

**Discussion by ripper121** [pschatzmann/ESP32-A2DP#309] [Q&A] [answered]
Is it possible to send a MP3 file from SD-Card?

**Comment by pschatzmann** [comment, marked as answer]
Have a look at the examples in the [arduino audio tools](https://github.com/pschatzmann/arduino-audio-tools)
However if you are a beginner, I do not recommend that you start with this...

I do not see this as an issue, so I convert it to a discussion..

**Reply by ripper121** [reply]
Im using the M5Stack:
https://gist.github.com/mongonta0716/18c28bff2dc533d8b51903c939a66a61
is it possible to merge it with this?
https://github.com/pschatzmann/ESP32-A2DP/blob/main/examples/bt_music_sender/bt_music_sender.ino

**Comment by pschatzmann** [comment]
I don't see the point: [bt_music_sender.ino](https://github.com/pschatzmann/ESP32-A2DP/blob/main/examples/bt_music_sender/bt_music_sender.ino) does not use any functionality which would be specific to the M5 stack and it should work w/o change...

In any case try first to to get aquainted with your board and try to make all the SD functionality work outside of my audio tools. This way you will learn about the different SD libraries and how to use them with your board.
